packages/shared/src/utils/debug/dbPerfMonitor.ts

```
@@ -348,6 +372,17 @@
     appStorageCallDetails[method].total += 1;
 
     globalRecentCalls.push([getNowString(), `${method}__${key}`]);
+
+    if (
+      shouldLocalDbDebuggerRule[`${method}__${key}`] &&
+      appStorageCallDetails[method].details[key] >=
+        shouldLocalDbDebuggerRule[`${method}__${key}`]
+    ) {
+      logResult();
+      if (settings?.debuggerEnabled) {
+        debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)